.travis.yml

```diff
@@ -21,11 +21,9 @@ env:
 matrix:
   include:
     - php: 5.6
-      env: DEPS=dev
+      env: DEPS=dev SYMFONY_VERSION=2.8.*
```

Comment:
if you change this value to `3.0.x-dev as 2.8.99`, we should be able to test against Symfony 3 (I used the same successuflly line in another project)